tensorzero-derive/tests/ui.rs

```diff
@@ -0,0 +1,5 @@
+#[test]
```

Comment:
I would call these API

Reply:
The standard terminology of macro output tests is 'ui tests', but I can add a comment